h2o-assemblies/main/build.gradle

```diff
@@ -23,6 +23,28 @@ dependencies {
     }
     compile project(":h2o-parquet-parser")
     compile "org.slf4j:slf4j-log4j12:1.7.5"
+
+    
+}
+
+//
+// Bundle additional modules
+// 
+if (project.hasProperty("withVisDataServer") && project.withVisDataServer == "true") {
```

Comment:
Just an idea - we could extract this to a file called `components.gradle` which would contain function `bundleAdditionalModules()` and we could call it here. 

or maybe have `gradle/components/componentsUtil.gradle` script with the mentioned method and then have several components with the code for bundling the component ( versions, dependencies ), such as `gradle/components/visualServer.component.gradle`, which would contain the code here. The function from the previous script could discover and apply all components.

The functionality would be the same, however, it would be better separated for future if there is need to have more optional modules